library/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.github.paolorotolo.appintro">
 
-    <application ></application>
+    <uses-permission android:name="android.permission.VIBRATE" />
```

Comment:
AFAIK This was intentional. We don't want to propage the `android.permission.VIBRATE` to all the apps that are using this library. If you (android dev) want to use the Vibration you need to define the permission in the app manifest and enable the feature.

Reply:
Solved on commit 0ff4cc7